Moderation-Policy.md

```diff
@@ -79,64 +78,73 @@ Note, however, that unfamiliarity with the [Code of Conduct][] does not excuse a
+Any Collaborator found to be violating the privacy of the nodejs/moderation repository by repeatedly sharing or discussing the details of nodejs/moderation issues in any public forum or social media service risks being permanently Removed from the Node.js GitHub organization.
 
-All Moderation-related decisions follow the normal [TSC motion and voting process](https://GitHub.com/nodejs/TSC/blob/master/TSC-Charter.md#section-8-voting).
+### Moderation Team
 
-TSC members directly involved in a Moderation issue -- as either the Requester or author of the Post in question -- are expected to excuse themselves from any decisions required to resolve the issue.
+The Node.js Moderation Team is tasked with enforcement of this policy.
 
-### Privacy of the nodejs/moderation Repository
+Moderation team members are Collaborators nominated by either the TSC or CommComm and must be approved by *both* committees. If there are objections to a specific nomination, then a simple majority vote of all TSC and CommComm members is required. Moderators are removed from the Moderation Team following the same process.
 
-The nodejs/moderation Repository is used to discuss the potentially sensitive details of any specific moderation issue. The repository is private but accessible to all Collaborators. The details of any issue discussed within the nodejs/moderation repository are expected to remain confidential and are not to be discussed in any public forum or social media service.
+Twice per month, the Moderation Team must provide a report of all Moderation actions taken by the Moderation Team to both the CommComm and TSC.
 
-Any Collaborator found to be violating the privacy of the nodejs/moderation repository by repeatedly sharing or discussing the details of nodejs/moderation issues in any public forum or social media service risks being removed from the Node.js GitHub organization through standard TSC motion and vote.
+#### Escalation of Issues
 
-### TSC Delegation of Authority to a "Moderation Working Group"
+Any Moderation issue or dispute may be escalated to the TSC and CommComm for review by tagging the original issue, pull request, or associated nodejs/moderation repository tracking issue with the `moderation-review` label. Any such Moderation action may be overturned through a simple majority vote of all TSC and CommComm members.
 
-For any Moderation issue that does not directly involve a TSC member, the TSC may choose to delegate some or all of it's Moderation-related responsibilities to a "Moderation Working Group". All members of such a Working Group must be Collaborators and the TSC will have responsibility for selecting the membership of that Moderation Working Group.
+Moderation Team members directly involved in a Moderation issue -- as either the Requester or author of the Post in question -- are expected to recuse themselves from any decisions required to resolve the issue.
```

Comment:
I would change "are expected to recuse themselves" to "are required to recuse themselves." I don't think that should be up for discussions or negotiation.